README.md

```diff
@@ -1 +1,36 @@
-# Awsome-books
\ No newline at end of file
+# Awesome books: plain JavaScript with objects
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio: setup and mobile version skeleton
+
+> The project is build a basic website that allows users to add/remove books from a list. It is done by using JavaScript objects and arrays. And also the DOM is dynamically modifying and added basic events.
+
+## Built With
+
+- HTML and CSS
+- GitHub and Figma
+
+## Author
+
+- GitHub: [GitHub](https://github.com/zmekonnen251)
+- Twitter: [Twitter](https://twitter.com/mek_zela)
+- LinkedIn: [Zelalem G. Mekonnen](https://www.linkedin.com/in/zelalem-getachew/)
```

Comment:
- [ ] I see that two developers are working on this project. Kindly add the second developer info to the readme file.